### PR TITLE
Fix edge command forwarder ordering

### PR DESCRIPTION
**Pull request by thjaeckle**
After #1416 we found a bug that the order of commands might be mixed up in `EdgeCommandForwarderActor` when applying the `SignalTransformer` in order to modify passed through signals.

As this is a regression (Ditto always ensures that for a single device/thing) all commands are processed in the same order they "arrive", this PR fixes the bug.

### CODE REVIEW

File: edge/service/src/main/java/org/eclipse/ditto/edge/service/dispatching/EdgeCommandForwarderActor.java [hunk 1]
```diff
@@ -151,25 +163,53 @@ public Receive createReceive() {
...
+                askWithRetryCommandForwarder.forwardCommand(command,
+                        shardRegions.things(),
+                        sender);
+            } else {
+                shardRegions.things().tell(transformed, sender);
+            }
+        }));
+    }
+
+    private void scheduleTask(final Signal<?> signal,
```
Comment (kalinkostashki) [inline edge/service/src/main/java/org/eclipse/ditto/edge/service/dispatching/EdgeCommandForwarderActor.java:188]: maybe rename signal var to thingSignal so readability is improved.

Comment (thjaeckle) [reply edge/service/src/main/java/org/eclipse/ditto/edge/service/dispatching/EdgeCommandForwarderActor.java:188]: Not only thing signals are passed into that method, but also policy signals, connectivity commands, etc.

**Comment by kalinkostashki** [comment]
Looks good to me :)

**Review by kalinkostashki — APPROVED** [review]
LGTM :)

**Comment by Yannic92** [comment]
This caused some bugs revealed in internal testing library. Reverting this with: https://github.com/eclipse/ditto/pull/1453

**Pull request merged**